src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -37,17 +37,18 @@ internal enum TreeRotation
     [DebuggerTypeProxy(typeof(ICollectionDebugView<>))]
     [DebuggerDisplay("Count = {Count}")]
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public partial class SortedSet<T> : ISet<T>, ICollection<T>, ICollection, IReadOnlyCollection<T>, ISerializable, IDeserializationCallback
     {
         #region Local variables/constants
 
-        private Node _root;
-        private IComparer<T> _comparer;
-        private int _count;
-        private int _version;
+        private Node root; // Do not rename (binary serialization)
```

Comment:
Yes but for some reason it didn't catch it before... This leads me to the conclusion that xunit Assert.Equals can't check equality for subitems on specialized objects like this one. Therefore I need to refine my tests and check all serializable types again if all variables are renamed.